security/wazuh-agent/src/opnsense/scripts/wazuh/setup.php

```diff
@@ -56,7 +56,11 @@
  * Temporary solution to link log files so we can view at least the last items in the file easily for debug purposes
  * It looks like ossec is not able to log to syslog directly, which means our log files live outside our normal bounds
  **/
-mkdir("/var/log/wazuhagent/ossec/", 0700, true);
```

Comment:
FYI @ also works ;)

Reply:
Indeed, a mkdir() failing is not the end of the world :) 
Would you prefer it? I can submit it again like that.

Reply:
The current fix is fine, no worries